tasks/build/verify_node_module_file_count.js

```diff
@@ -0,0 +1,16 @@
+const glob = require('glob');
+
+const EXPECTED_MODULE_COUNT = 1145;
+
+module.exports = function (grunt) {
+  grunt.registerTask('_build:verifyNodeModuleFileCount', function () {
```

Comment:
Can you extract this stuff into a separate PR? This will fail whenever someone adds a package etc, so I think it's something we have to think closely about.

Reply:
Yeah, totally meant for it to be separate